app/service/utils.go

```diff
@@ -9,9 +9,12 @@ import (
 
 // QueryParamToInt64Slice extracts from the query parameter of the request a list of integer separated by commas (',')
 func QueryParamToInt64Slice(req *http.Request, paramName string) ([]int64, error) {
-
-	idsStr := strings.Split(req.URL.Query().Get(paramName), ",")
-	var ids []int64
+	ids := []int64{}
```

Comment:
I would rather prefer `var ids []int64` (see here https://github.com/golang/go/wiki/CodeReviewComments#declaring-empty-slices)

Reply:
it is not the same. with `var ids []int64` I get a nil while I want to start with an empty array

Reply:
Nil is not a problem https://play.golang.org/p/nUbbKcKhxaG

Reply:
https://play.golang.org/p/LRoPTFtNurN

Reply:
But for my function, it is. My function is returning an empty list (as tested and as used) when there is no IDs. If I use nil at this line, it might return nil in some cases with is not expected (in tests, by caller)